src/chttpd_cors.erl

```diff
@@ -306,6 +316,35 @@ get_cors_config(#httpd{cors_config = undefined}) ->
 get_cors_config(#httpd{cors_config = Config}) ->
     Config.
 
+cors_config(Host, Key, Default) ->
+    config:get(cors_section(Host), Key,
+                     config:get("cors", Key, Default)).
+
+cors_section(Host0) ->
+    {Host, _Port} = split_host_port(Host0),
```

Comment:
> Speaking of, vhost's cors config in default.ini mentions that scheme must to be included, are we dropping this req?

We use [string:rchr/2](https://github.com/apache/couchdb-chttpd/pull/101/files#diff-67787f983e20e7ee8910d35365d48a42R329) to find out the position of right most occurrence of `:`. Then we split at that position preserving the schema. 

This actually answers your previous question as well:
> I don't get it, why not just string:tokens/2 here?